packages/near-membrane-base/src/blue.ts

```diff
@@ -161,52 +153,99 @@ export function blueProxyFactory(env: MembraneBroker) {
         // istanbul ignore else
         if ('writable' in blueDescriptor) {
             // We are dealing with a value descriptor.
-            const { value } = blueDescriptor;
-            blueDescriptor.value =
-                typeof value === 'function'
```

Comment:
unfolding these optimizations since `getBlueValue` will also do a typeof, seems unnecessary, but I don't have a way to validate that this is in fact faster :), going with my gut feeling at the moment on this one.